config/config.json

```diff
@@ -404,7 +407,8 @@
         "titleBlockUrlListApproch":"Allow everything except certain distracting sites",
         "allowUrlsQuestionText":"Which URLs do you want to allow?",
         "blockUrlsQuestionText":"Which URLs do you want to block?",
-        "enterMultiUrlText":"Enter single or multiple urls(one in each line)"
+        "enterMultiUrlText":"Enter single or multiple urls(one in each line)",
+        "meetingsFocusInfoText":"Meetings focus mode helps restricit use of certain apps when you are attending a meeting. It can block everything else to help you concentrate."
```

Comment:
The meetings focus mode is used to detect when you are in a meeting so you don't get asked to take a break when you're in an important meeting. Add the apps and websites you use for video conferencing.

Reply:
@manishjain999 
https://focusbear.slack.com/archives/C04F3DHF4N5/p1675661578958239